app/interfaces/api/v1/dropdown_data.rb

```diff
@@ -148,6 +148,12 @@ def unique_code
           desc 'Return all Transfer Case Conclusions'
           get { present ::Claim::TransferBrain::CASE_CONCLUSIONS.to_a, with: API::Entities::SimpleKeyValueList }
         end
+
+        resource :case_stages do
+          desc 'Return all Case Stages'
+          params { use :role_filter }
+          get { present CaseStage.__send__(role), with: API::Entities::CaseStage }
```

Comment:
One minor tweak.
```suggestion
          get { present CaseStage.active.__send__(role), with: API::Entities::CaseStage }
```
Joel only added the scope late last week, it will prevent showing obsolete responses, e.g.
```json
[
  {
    "case_type_id": 10,
    "unique_code": "OBSOLETE1",
    "description": "Pre PTPH (evidence served)",
    "roles": [
      "lgfs"
    ]
  },
  {
    "case_type_id": 8,
    "unique_code": "OBSOLETE2",
    "description": "Pre PTPH (no evidence served)",
    "roles": [
      "lgfs"
    ]
  },
  {
    "case_type_id": 6,
    "unique_code": "PREPTPHADJ",
    "description": "Pre PTPH or PTPH adjourned",
    "roles": [
      "lgfs"
    ]
  }
]
```